Services/RequestServer/ConnectionFromClient.cpp

```diff
@@ -519,9 +521,22 @@ void ConnectionFromClient::check_active_requests()
         if (msg->msg != CURLMSG_DONE)
             continue;
 
-        ActiveRequest* request = nullptr;
-        auto result = curl_easy_getinfo(msg->easy_handle, CURLINFO_PRIVATE, &request);
+        void* application_private = nullptr;
+        auto result = curl_easy_getinfo(msg->easy_handle, CURLINFO_PRIVATE, &application_private);
         VERIFY(result == CURLE_OK);
+        VERIFY(application_private != nullptr);
+
+        // FIXME: Come up with a unified way to track websockets and standard fetches instead of this nasty tagged pointer
+        if (reinterpret_cast<uintptr_t>(application_private) & 1) {
```

Comment:
This C-goo is only required because I was too lazy to set up a new multi handle for web sockets explicitly. Open to doing other strats in a follow-up if you have good ideas, CxBoog

Reply:
This honestly seems fine, maybe name that bit as a flag?
could also allocate a Variant on the heap I guess, if you want alternatives.